ghost/stack.yml

```diff
@@ -17,6 +17,7 @@ services:
       database__connection__user: root
       database__connection__password: example
       database__connection__database: ghost
+      url: example.org
```

Comment:
I believe this format is wrong -- this should be a full base URL, including protocol and port (as in the examples linked at the top of this section of `stack.yml`)